HerPublicWebsite/Models/Questionnaire/AddressViewModel.cs

```diff
@@ -1,7 +1,9 @@
+using GovUkDesignSystem.Attributes.ValidationAttributes;
+
 namespace HerPublicWebsite.Models.Questionnaire;
 
-public class AddressViewModel: QuestionFlowViewModel
+public record AddressViewModel: QuestionFlowViewModel
```

Comment:
Why switch these to records?

Reply:
I think records are better for data types generally, and ToString gets implemented automatically on them, ViewModels probably shouldn't be though